### ISSUE TITLE
Error parsing apk

### ISSUE THREAD

**Issue by icewyvrn** [vamsi3/screen-translator#2]
I can't install it on Android 11 because of the problem parsing the apk.

**Comment by vamsi3** [comment]
Hi @icewyvrn, I have released an update to decrease the min Android version requirement, so `v0.0.3` `apk` should work fine for you. Let me know if there is any further issue with it.

**Comment by icewyvrn** [comment]
Thank you. It works, but the notification panel does not collapse when the quick tile is triggered.

**Comment by vamsi3** [comment]
Can you open the App and check if the notification panel swipe is enabled? Also do adjust the speed of the swipe according to animation speed of your phone (you can start from high `ms` and go low `ms`)

**Comment by icewyvrn** [comment]
Yeah, it was disabled. I enabled it, and the notification panel collapses now. Thank you!

**Comment by vamsi3** [comment]
Instead of mimicking a swipe-up gesture, I plan to dismiss the notification shade more systematically using Android APIs provided. I'll release an update soon on this.

Closing this issue for now. (Please do let me know any issues/feature requests, I'd be more than happy to know)

**Comment by vamsi3** [comment]
@icewyvrn You can check out the `v0.0.5` release. Do remember to adjust the new delay setting. Also, I want to highlight that this setting replaces the previous notification swipe-up duration setting.

This new setting sets the time you want to wait after the notification shade collapse before taking the screenshot. Play with the settings as you wish until I get this right out of the box. Once you get the settings right, it works well forever!

**Comment by icewyvrn** [comment]
Hello, the new update is broken. The translate app does not open anymore.

**Comment by vamsi3** [comment]
Oh, It works well on my side. I've been using it since the release. Let me check that out. I could be probably something related to my code specific to Android 11.
